src/components/Login.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import auth from '../auth/auth';
+
+interface OverviewProps extends RouteComponentProps<{name:string}> {
+
+}
+
+function Login(props:OverviewProps) {
+  return (
+    <div>
+      Please Log in
+      <form>
+        <input type="text" placeholder="cactus" />
+        <input type="text" placeholder="****" />
+      </form>
+      <button
+        type="button"
+        onClick={() => {
```

Comment:
it might be better to have this as a separately declared function in case we need more for the login form